docs/siem/detections/api/exceptions/exceptions-api-overview.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[exceptions-api-overview]]
+== Exceptions API
+
+Exceptions are associated with detection and endpoint rules, and are used to
+prevent a rule from generating an alert from incoming events even when the
+rule's other criteria are met. They can be used to reduce the number of false
+positives, and to prevent trusted processes and network activity from
+generating unnecessary alerts.
+
+Exceptions are made up of:
+
+* *Exception containers*: A container for related exceptions. In general, a
+single exception container contains all the exception items relevant for
+a subset of rules. For example, a container can be used to group together
+network-related exceptions that are relevant for a large number of network
+rules. The container can then be associated with all the relevant rules.
+* *Exception items*: The query (fields, values, and logic) used to prevent
+rules from generating alerts. When an exception item's query evaluates to
+`true`, the rule does not generate an alert.
```

Comment:
Not sure if you want to repeat the explanation of items getting `OR` here or not. I know you had it up above, but it was such a great explanation, figured it might be useful here?

Reply:
@yctercero - hopefully the diagram under this makes things clear. Can you take a look at the preview and let me know: https://security-docs_35.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-overview.html